### PR TITLE
Extra tags from files

### PR DESCRIPTION
**Pull request by alexopus**
- Add edit field to image_selector to add tags, which will be stored in the EXIF data.
- Grab these tags when scheduling to Deviant.

### CODE REVIEW

File: src/clients/test.py [hunk 1]
```diff
@@ -7,8 +7,8 @@ class TestClient:
     def __init__(self, account: Account):
         self.account = account
 
-    def schedule(self, image_path, caption):
-        print(f"Received in schedule args: {image_path}, {caption}")
+    def schedule(self, image_path, caption, content_tags):
```
Comment (nickve28) [inline src/clients/test.py]: If they aren't used here yet, I prefer keeping it out

File: tests/utils/image_metadata_test.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import unittest
+import os
+from PIL import Image
+import piexif
+from utils.image_metadata_adjuster import ImageMetadataAdjuster
+
+
+def create_test_image(path):
```
Comment (nickve28) [inline tests/utils/image_metadata_test.py:8]: 👍

**Review by nickve28 — APPROVED** [review]

File: src/clients/test.py (hunk 1 above)
Comment (alexopus) [reply src/clients/test.py]: ok, i threw it out from unused places.

File: src/image_selector.py [hunk 3]
```diff
@@ -43,13 +43,14 @@ def post_now(filename, mode):
     account_copy = select_account(account_data, scheduler_profile_ids=scheduler_profile_ids)
     account_copy.set_config_for(filename)
     caption = ImageMetadataAdjuster(filename).get_caption() or ""
+    content_tags = ImageMetadataAdjuster(filename).get_content_tags() or ""
 
     response = False
     if mode == "Twitter":
-        response = TwitterClient(account_copy).schedule(filename, caption)
+        response = TwitterClient(account_copy).schedule(filename, caption, content_tags)
```
Comment (nickve28) [inline src/image_selector.py]: will this work? twitter doesn't have this arg assigned anymore

Comment (alexopus) [reply src/image_selector.py]: Ah yes, missed that, thx.

**Review by nickve28 — APPROVED** [review]

**Pull request merged**